modules/logging-in-by-using-the-web-console.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="logging-in-by-using-the-web-console_{context}"]
+= Logging in to the cluster by using the web console
+
+You can log into your cluster as `kubeadmin` by using the {product-title} web console after the Console Operator becomes available.
+
+.Prerequisites
+
+* You have access to the installation host.
+* The Console Operator is available on your cluster.
```

Comment:
Would the Console Operator ever not be installed?

Reply:
Yes. After a UPI installation there might be a requirement to approve pending certificate signing requests. Until those are approved, the Console Operator might be unavailable. If that is the case, then the console will not be available. I experienced this in my testing.

Thank you for highlighting this, I have made need for the prerequisite clearer by adding the following note after that line:

[NOTE]
====
You might need to approve some certificate signing requests (CSRs) after a user-provisioned infrastructure installation. The Console Operator might be unavailable until some CSRs are approved. You cannot access the {produtct-title} web console unless the Console Operator is available.
====

Reply:
I'd probably make the prereq something like, "You have completed cluster installation, and all cluster operators are available." I agree that adding the note with more context would be confusing for the IPI installations, but that a little more info might be useful for UPI. (And that if we need to add more info about the CSRs and core Operator availability, that the UPI installation modules/assemblies themselves are the better place for it.)

Reply:
That's good advice, thank you!

I have a task in my proposal document to add a note in the 'Approving the CSRs for your machines' section in the UPI installation procedure, which will meet this requirement.

I have now updated the intro to this module to remove the reference to the console operator specifically. I have also changed the prerequisite accordingly.